### PR TITLE
Figma Design Changes 2

### PR DESCRIPTION
**Pull request by constrictingsnake**
…flect the change

# Pull Request

## Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Refactoring (no functional changes)
- [ ] Performance improvement
- [ ] Other (please describe):

### Code Review Preparation
- [ ] I have performed a self-review and reviewed the changed files
- [ ] My code follows the project's style guidelines and passes linting
- [ ] I have added comments in hard-to-understand areas and updated documentation

### Testing (Ignore for the first PR)
- [ ] I have added relevant tests and they pass locally
- [ ] I have tested the changes manually, including edge cases

### Code Quality & Security
- [ ] I have formatted my code and removed debugging/unused code
- [ ] No TypeScript errors or linting issues
- [ ] No sensitive information committed (API keys, passwords, etc.)
- [ ] User inputs are validated and sanitized

### Dependencies & Deployment
- [ ] Dependencies are documented and package-lock.json is updated if needed
- [ ] Changes work with Firebase emulator and environment variables are documentedRetry

### CODE REVIEW

File: app/debug/data/page.tsx [hunk 1]
```diff
@@ -1368,6 +1341,8 @@ const CLIENT_NOTES = [
 ];
 export default function page() {
     const btnClass = "border border-gray-400 rounded px-6 py-2 hover:bg-gray-100 text-sm w-64";
+    const { changes, isLoading } = useWarehouseHistory();
```
Comment (joelchem) [inline app/debug/data/page.tsx]: why is this even here

**Review by joelchem — CHANGES_REQUESTED** [review]
fix the leftover stuff

File: app/debug/data/page.tsx (hunk 1 above)
Comment (constrictingsnake) [reply app/debug/data/page.tsx]: bro i was doing something
